backend/models/seller.py

```diff
@@ -1,18 +1,20 @@
-class Seller:
-    __fullname: str
-    __phone: str
-    __email: str
-    __id: int
+from sqlalchemy import Column, String
+from backend.models.base_model import BaseModel
 
-    def __init__(self, fullname: str, phone: str, email: str, id: int = None):
-        self.__fullname = fullname
-        self.__phone = phone
-        self.__email = email
-        self.__id = id
+class Seller(BaseModel):
+    __tablename__ = 'seller'
+    __name = Column('name', String(length=100) )
+    __phone = Column('phone', String(length=18) )
+    __email = Column('email', String(length=30) )
```

Comment:
Aqui não tem porquê utilizar variáveis privadas com property, é melhor utilizar públicas mesmo, visto que se for necessária uma validação, será feita com o `@validates` e se precisar de tratamento será feito no dao.